instrumentation/spring/spring-webmvc-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webmvc/v5_3/HttpRouteSupport.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.webmvc.v5_3;
```

Comment:
I also changed the package name, in accordance with #932

Reply:
I had to think through this (https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/932#issuecomment-1206678184), but I agree